README.md

```diff
@@ -73,7 +73,7 @@ OSM is under active development and is **NOT** ready for production workloads.
 | TCPRoute | tcproutes.specs.smi-spec.io | [v1alpha4](https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-specs/v1alpha4/traffic-specs.md#tcproute) | |
 | UDPRoute | udproutes.specs.smi-spec.io | _not supported_ | |
 | TrafficSplit | trafficsplits.split.smi-spec.io | [v1alpha2](https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-split/v1alpha2/traffic-split.md) | |
-| TrafficMetrics  | \*.metrics.smi-spec.io | [v1alpha1](https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-metrics/v1alpha1/traffic-metrics.md) | 🚧 **In Progress** [#379](https://github.com/openservicemesh/osm/issues/379) 🚧 |
+| TrafficMetrics  | \*.metrics.smi-spec.io | [v1alpha1](https://github.com/servicemeshinterface/smi-spec/blob/v0.6.0/apis/traffic-metrics/v1alpha1/traffic-metrics.md) | |
```

Comment:
That issue was closed because there's no more work for us to do here for SMI Metrics, but there's still a PR open to add OSM to SMI's metrics server that I think should get merged before we can definitively say OSM supports SMI Metrics: https://github.com/servicemeshinterface/smi-metrics/pull/77.

For that reason I'd say let's at least keep the status of SMI Metrics in the README as "In Progress" and either remove the link to the issue or reopen it to make that less ambiguous.

Reply:
"In Progress" status is good as the issue links can become obsolete, will update the same thanks @nojnhuh

Reply:
@shashankram @nojnhuh is "In Progress" status good to go?